yarpl/include/yarpl/flowable/EmitterFlowable.h

```diff
@@ -208,7 +208,7 @@ class EmitterWrapper : public EmiterBase<T>, public Flowable<T> {
       : emitter_(std::move(emitter)) {}
 
   void subscribe(Reference <Subscriber<T>> subscriber) override {
-    new EmiterSubscription<T>(get_ref(this), std::move(subscriber));
+    make_ref<EmiterSubscription<T>>(get_ref(this), std::move(subscriber));
```

Comment:
The code here is functionally identical; the reason EmitterSubscription doesn't get immediately destroyed is because we call `subscriber->onSubscribe(make_ref(this))` in `EmitterSubscription`'s ctor (so the subscriber keeps a reference to it)